70_FLIPFluids/app_resources/compute/diffusion.comp.hlsl

```diff
@@ -0,0 +1,142 @@
+#include "../common.hlsl"
+#include "../gridUtils.hlsl"
+#include "../cellUtils.hlsl"
+#include "../descriptor_bindings.hlsl"
+
+struct SPushConstants
+{
+    float4 diffusionParameters;
+};
+
+[[vk::push_constant]] SPushConstants pc;
```

Comment:
scalar layout also applies to push constants, you can make the params a `float3`

Reply:
ah all 4 are used, but no name that makes sense